packages/reactivity/src/reactive.ts

```diff
@@ -147,8 +164,32 @@ export type DeepReadonly<T> = T extends Builtin
   : Readonly<T>
 
 /**
- * Creates a readonly copy of the original object. Note the returned copy is not
- * made reactive, but `readonly` can be called on an already reactive object.
+ * Creates a readonly copy of the original object.
+ *
+ * All (deep) properties within the proxy object will be readonly, too. As with
+ * `reactive`, all (deep) refs will be unwrapped. The returned object will not
+ * be reactive by default, but already-reactive objects can be passed to
+ * `readonly`.
```

Comment:
Probably worth just reusing tha API docs with a reference to `shallowReactive()` for this one